apps/schools/domains/query/components/currentQuery/index.tsx

```diff
@@ -122,6 +192,18 @@ export const CurrentQuery = () => {
                                     {query?.additional.text || 'Не определен'}
                                 </div>
                             </div>
+                            {history?.results.length !== 0 && (
+                                <div className={styles.queryHistory}>
+                                    <div className={styles.queryHistoryHeader}>История изменения заявки</div>
+                                    {!historyLoading ? (
+                                        <div className={styles.queryHistoryScrollContainer}>{formattedHistory}</div>
+                                    ) : (
+                                        <>
+                                            <Spin></Spin>
+                                        </>
+                                    )}
+                                </div>
```

Comment:
why did you start using isFetching somewhere, but still isLoading somewhere?

Reply:
isLoading triggers only on first request but isFetching on every request including refetch

Reply:
somewhere we can use isLoading, somewhere - isFetching
in my mind there is no problem

Reply:
https://lamart.atlassian.net/browse/SCHOOL-794